src/AutoRest.CSharp/Common/Generation/Writers/CodeWriterExtensions.cs

```diff
@@ -228,7 +228,7 @@ public static CodeWriter WriteReferenceOrConstant(this CodeWriter writer, Refere
             this CodeWriter writer,
             Action<CodeWriter, CodeWriterDelegate> valueCallback,
             ObjectType objectType,
-            ObjectTypeConstructor constructor,
+            ObjectTypeConstructor? constructor,
```

Comment:
Why do we need the ability to pass null here? There should be a ctor that we are calling.

Reply:
no longer needed